docs/modules/publisher-directmethods.md

```diff
@@ -65,46 +61,48 @@ The `_V1` direct methods  uses the  payload schema as described below:
+| Attribute                       | Mandatory | Type    | Default | Description                                                                                                                                                                                                                                                                                                       |
+| ------------------------------- | --------- | ------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Id`                            | Yes       | String  | N/A     | The OPC UA NodeId in the OPC UA server whose data value changes should be published. <br>Can be specified as NodeId or ExpandedNodeId <br>as per OPC UA specification, or as ExpandedNodeId IIoT format <br>{NamespaceUi}#{NodeIdentifier}.                                                                                                               |
+| `ExpandedNodeIdId`              | No        | String  | null    | Enables backwards compatibility. Must be <br>specified as ExpandedNodeId as per OPC UA specification.                                                                                                                                                                                                      |
+| `OpcSamplingInterval`           | No        | Integer | 1000    | The sampling interval for the monitored item to be <br>published. Value expressed in milliseconds. The value is used as defined in the OPC UA specification. Ignored when OpcSamplingIntervalTimespan is present.                                                                                                                                                                                                                 |
+| `OpcSamplingIntervalTimespan`   | No        | String  | null    | The sampling interval for the monitored item to be <br>published. Value expressed in Timespan <br>string({d.hh:mm:dd.fff}). <br>The value is used as defined in the OPC UA specification.                                                                                                                                     |
+| `OpcPublishingInterval`         | No        | Integer | 1000    | The publishing interval for the monitored item to be <br>published. Value expressed in milliseconds. <br>This value will be overwritten when a publishing interval <br>is explicitly defined in the DataSetWriter for the specified node. The value is used as defined in the OPC UA specification. Ignored when OpcPublishingIntervalTimespan is present.                                                                            |
```

Comment:
>This value will be overwritten when a publishing interval <br>is explicitly defined in the DataSetWriter for the specified node. [](http://example.com/codeflow?start=173&length=128)

same as comment on next line.

Reply:
Changed wording to indicate that `OpcPublishingInterval` will overwrite value of `DataSetPublishingInterval` for the specified node. Please have a look.